src/utils/jump_utils.jl

```diff
@@ -29,50 +29,56 @@ function jump_value(input::Float64)::Float64
     return input
 end
 
-function to_matrix(array::DenseAxisArray)
+function to_matrix(array::DenseAxisArray{T, 1, K}) where {T, K <: NTuple{1, Any}}
+    data = jump_value.(array.data)
+    data = reshape(data, length(data), 1)
+    return data
+end
+
+function to_matrix(array::DenseAxisArray{T, 1, K}) where {T <: Real, K <: NTuple{1, Any}}
+    data = reshape(deepcopy(array.data), length(array.data), 1)
+    return data
+end
+
+function to_matrix(array::DenseAxisArray{T, 2, K}) where {T, K <: NTuple{2, Any}}
     ax = axes(array)
-    len_axes = length(ax)
-    if len_axes == 1
-        data = jump_value.((array[x] for x in ax[1]))
-    elseif len_axes == 2
-        data = Matrix{Float64}(undef, length(ax[2]), length(ax[1]))
-        for t in ax[2], (ix, name) in enumerate(ax[1])
-            data[t, ix] = jump_value(array[name, t])
-        end
-        # TODO: this needs a better plan
-        #elseif len_axes == 3
-        #    extra_dims = sum(length(axes(array)[2:(end - 1)]))
-        #    arrays = Vector{Matrix}()
-
-        #    for i in ax[2]
-        #        third_dim = collect(fill(i, size(array)[end]))
-        #        data = Matrix{Float64}(undef, length(last(ax)), length(first(ax)))
-        #        for t in last(ax), (ix, name) in enumerate(first(ax))
-        #            data[t, ix] = jump_value(array[name, i, t])
-        #        end
-        #        push!(arrays, data)
-        #    end
-        #    data = vcat(arrays)
-    else
-        error("array axes not supported: $(axes(array))")
+    data = Matrix{Float64}(undef, length(ax[2]), length(ax[1]))
+    for t in ax[2], (ix, name) in enumerate(ax[1])
+        data[t, ix] = jump_value(array[name, t])
     end
-
     return data
 end
 
-# TODO: These functions could be use in other places for the store etc
+# to_matrix functions are used to convert JuMP.Containers to matrices that can be written into
+# HDF5 Store.
+function to_matrix(array::DenseAxisArray{T, 2, K}) where {T <: Real, K <: NTuple{2, Any}}
+    return deepcopy(permutedims(array.data))
+end
+
+function to_matrix(::DenseAxisArray{T, N, K}) where {T, N, K <: NTuple{N, Any}}
+    throw(
```

Comment:
Delete the throw.

Reply:
fixed via 43da263767bf544e0ff4f3b9d861c3a9c36050c6